src/commands/createNewProject/validateFunctionProjects.ts

```diff
@@ -152,19 +152,20 @@ async function promptToUpdateDebugConfiguration(fsPath: string): Promise<boolean
 
 async function verifyPythonVenv(projectLanguage: string | undefined, folderPath: string, actionContext: IActionContext): Promise<void> {
     if (projectLanguage === ProjectLanguage.Python) {
-        if (!await fse.pathExists(path.join(folderPath, funcEnvName))) {
+        const venvName: string | undefined = getFuncExtensionSetting(pythonVenvSetting, folderPath);
+        if (venvName && !await fse.pathExists(path.join(folderPath, venvName))) {
             actionContext.properties.pythonVenvExists = 'false';
 
             const settingKey: string = 'showPythonVenvWarning';
             if (getFuncExtensionSetting<boolean>(settingKey)) {
                 const createVenv: vscode.MessageItem = { title: localize('createVenv', 'Create virtual environment') };
-                const message: string = localize('uninitializedWarning', 'Failed to find Python virtual environment, which is required to debug and deploy your Azure Functions project.');
+                const message: string = localize('uninitializedWarning', 'Failed to find Python virtual environment "{0}", which is required to debug and deploy your Azure Functions project.', venvName);
```

Comment:
What will the venvName be if there is no pythonVenv setting?  Undefined?

Reply:
We won't get here if thats the case. `if (venvName && ` above

Reply:
Oh right, sorry.  My actual question is do we not prompt the user to create a venv if there's no setting detected?

Reply:
Correct we do not prompt. Seems fair to me, but let me know if you disagree

Reply:
Well, when a user creates via the Func CLI, they definitely won't have the pythonVenv setting.  Since at that time, the user can select a `projectLanguage`, they will be able to hit this check.  

`ensureVenv()` is only invoked at project creation so the only way they would be able to get to the Quickpick of selecting a venv is if they created a new project in a folder that already has venvs, right?

It seems like this check wouldn't catch the case of creating via Func CLI then, it'd only help if they cloned a project that already has the proper workspace setting.

Reply:
If the user creates through func cli - they will hit this dialog:
![image](https://user-images.githubusercontent.com/11282622/47176180-c890ea80-d2c9-11e8-9910-30dbb3dfe8d4.png)

Which creates tasks.json/settings.json/all that jazz including executing `ensureVenv`

`verifyPythonVenv` is only meant to handle the case where they clone a repo already configured for VS Code (but that doesn't have the venv since it's not checked in to git)

Reply:
Ah, I rest all further pressing the witness, your honor